core/server/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -204,8 +205,13 @@ public void stop() throws IOException {
   }
 
   @Override
```

Comment:
is there a guarantee that these methods will be never called concurrently with journal operations?